warehouse/models/mart/gtfs/_mart_gtfs.yml

```diff
@@ -95,6 +95,29 @@ models:
         description: '{{ doc("column_zipfile_files") }}'
       - name: zipfile_dirs
         description: '{{ doc("column_zipfile_dirs") }}'
+  - name: fct_vehicle_locations
+    description: |
+      De-duped vehicle positions, removing redundant/duplicated positions
+      in the underlying messages. Unique at the url/timestamp/vehicle/trip
+      level.
+    columns:
+      - name: key
+        description: |
+          Synthetic primary key.
+        tests:
+          - unique:
+              # TODO: make an RT test macro that does this date subtraction
+              # test hour = 0 UTC because that's 5pm Pacific = PM peak, good sample of data
+              config:
```

Comment:
do you need `config`? I didn't in mine to add a `where` to the uniqueness test

Reply:
I think `config:` is the new thing going forward but top-level `where:` is backwards-compatible?

The docs do seem aligned on `config:` https://docs.getdbt.com/reference/test-configs#test-specific-configurations